convex/game/conversationMembers.ts

```diff
@@ -12,11 +13,17 @@ export const conversationMembers = defineTable({
     v.object({ kind: v.literal('invited') }),
     v.object({ kind: v.literal('walkingOver') }),
     v.object({ kind: v.literal('participating'), started: v.number() }),
-    v.object({ kind: v.literal('left'), started: v.optional(v.number()), ended: v.number() }),
+    v.object({
+      kind: v.literal('left'),
+      started: v.optional(v.number()),
+      ended: v.number(),
+      with: v.id('players'),
+    }),
   ),
 })
   .index('conversationId', ['conversationId', 'playerId'])
-  .index('playerId', ['playerId', 'status.kind']);
+  .index('playerId', ['playerId', 'status.kind', 'status.ended'])
```

Comment:
one bit of Convex feedback: I am inferring here that we're adding `status.ended` so that we can have them ordered in that way, but it's not clear at the index usage site what the ordering will be, without finding this definition.